cmd/inspect.go

```diff
@@ -183,6 +209,58 @@ func populateNamespaces(out io.Writer, n map[string][]string) error {
 	return nil
 }
 
+func populateAnnotations(out io.Writer, as []*ast.AnnotationsRef) error {
+	// TODO: Sort by path, then location, for stable output
+	if len(as) > 0 {
+		fmt.Fprintln(out, "ANNOTATIONS:")
+		for _, a := range as {
+			fmt.Fprintln(out, "Path:", a.Path.String())
+			fmt.Fprintln(out, "Location:", a.Location.String())
+			if len(a.Annotations.Title) > 0 {
+				fmt.Fprintln(out, "Title:", a.Annotations.Title)
+			}
+			if len(a.Annotations.Description) > 0 {
+				fmt.Fprintln(out, "Description:", a.Annotations.Description)
+			}
+			if len(a.Annotations.Organizations) > 0 {
+				fmt.Fprintln(out, "Organizations:")
+				for i, o := range a.Annotations.Organizations {
+					fmt.Fprintf(out, " %d. %s\n", i+1, o)
+				}
+			}
+			if len(a.Annotations.Authors) > 0 {
+				fmt.Fprintln(out, "Authors:")
+				for i, a := range a.Annotations.Authors {
+					fmt.Fprintf(out, " %d. %s\n", i+1, a.CompactString())
+				}
+			}
+			if len(a.Annotations.Schemas) > 0 {
+				// NOTE(johanfylling): The Type Checker will MERGE all applicable schema annotations for a rule
+				// into one list. Here, child nodes OVERRIDE parent nodes' schema annotations instead (default annot. behavior).
+				// Should the former behavior be replicated here?
```

Comment:
That's an interesting question. Assuming that people will want to make sense of the output here, they wouldn't have a full picture if we didn't present all applicable schema annotations, would they? I don't have an example just now...

Perhaps we could make do with a warning here ("applicable schemas can be defined in parent nodes and will be merged for type checking" or something like that) if including all the relevant schemas is too cumbersome -- we don't really know if and how this is going to get used by our schema aficionados, do we?

Reply:
It could also be argued that we should merge all list-type annotations like this. E.g. The package has two authors, and a third author adds a single rule, which is annotated. Should the set of extracted annotations for the rule show only the one author or all three? 
That wouldn't work as naturally for `related_resources`, though; where you'd likely expect a list specific to the rule. I'd like to avoid having different merge strategies for different annotations; but at the same time, it should also make sens case-to-case ..

Reply:
Hmm would it make sense to not apply any merge logic, but instead display everything that's relevant? E.g. you may query for the annotations of `data.pack.p`, but since `data.pack`'s annotations are relevant, they will _also_ be shown, but in a direct way. I.e. expanding the selection, not merging the results. WDYT?

Reply:
That is definitely an option. Internally we generate an `AnnotationSet` that is ostensibly a tree view of unmerged annotations. A developer can create this given a list of `Module`s. For `inspect`, I'm thinking we should allow you to toggle between a flattened view of this tree, and an expanded view, where annotations are merged into all relevant packages/rules. This might not be entirely necessary, though.
The merge strategy for the latter, expanded view should be the same strategy used for the `self` keyword. For `self` I don't think a list of unmerged annotations is as useful as one merged set of annotations.